src/olympia/devhub/templates/devhub/versions/list.html

```diff
@@ -220,6 +220,7 @@
   <div class="item" id="version-list"
        data-stats="{{ url('devhub.versions.stats', addon.slug) }}">
     <div class="item_wrapper">
+      {% if addon.status != amo.STATUS_DISABLED %}{# i.e. Admin disabled #}
```

Comment:
We re-use the status codes for both Addon and File currently and STATUS_DISABLED doesn't mean admin disabled for File.   They should be separated (and I did that somewhat with the preliminary work) but that's a whole other issue.